server/lib/schema/index.ts

```diff
@@ -297,7 +297,6 @@ export const workspaceType = makeObjectType(
 
 import { UserActivityType } from "./UserActivity";
 import { WorkspaceActivityType } from "./WorkspaceActivity";
-import { generateMaliciousQuestionValue } from "../models/helpers/generateMaliciousQuestionValue";
```

Comment:
Nice catch. Kinda surprised Travis CI didn't notice this? If you wanted to take a look at the Travis CI setup, or recommend any other kind of tooling like some sort of automated linting, I think that would be valuable.

Reply:
Hmm haven't looked into automated linting outside of my IDE. Could be interesting to take a look next week, time permitting

Reply:
Cool, no rush. Something to keep in mind when we're thinking about next task to add to queue though.